Cargo.toml

```diff
@@ -28,7 +28,7 @@ typst-syntax = { path = "crates/typst-syntax", version = "0.12.0" }
 typst-timing = { path = "crates/typst-timing", version = "0.12.0" }
 typst-utils = { path = "crates/typst-utils", version = "0.12.0" }
 typst-assets = { git = "https://github.com/typst/typst-assets", rev = "5c0dcc0" }
-typst-dev-assets = { git = "https://github.com/typst/typst-dev-assets", rev = "b07d156" }
+typst-dev-assets = { git = "https://github.com/mkorje/typst-dev-assets", rev = "071ff36" }
```

Comment:
Pointing at a fork of assets is fine for dev work, but it shouldn't be in a PR marked as ready. I suggest at least marking the PR as draft status so it can't accidentally be merged with a snafu like this still in it. You can set draft status from the sidebar.